prime-router/metadata/fhir_mapping/hl7/codesystem/ExtensionUrlMapping.yml

```diff
@@ -190,4 +190,7 @@
   url: "https://reportstream.cdc.gov/fhir/StructureDefinition/placer-assigned-identifier"
 # Filler assigned identifier
 - id: "filler-assigned-identifier"
-  url: "https://reportstream.cdc.gov/fhir/StructureDefinition/filler-assigned-identifier"
\ No newline at end of file
+  url: "https://reportstream.cdc.gov/fhir/StructureDefinition/filler-assigned-identifier"
+# Producer's ID
```

Comment:
Change this comment to just sub ID.  It is generic enough to be use in multiple places

Reply:
fixed